packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -43,6 +43,13 @@ export interface BundlingOptions {
    */
   readonly volumes?: DockerVolume[];
 
+  /**
+   * Where to mount the specified volumes from
+   * Docker [volumes-from option](https://docs.docker.com/engine/reference/commandline/run/#mount-volumes-from-container---volumes-from)
+   * @default - no volumes-from options
+   */
+  readonly volumesFrom?: string;
```

Comment:
Any reasons this isn't an array?
```suggestion
  readonly volumesFrom?: string[];
```

Reply:
updated